content/blog/talk-to-youtube-videos-with-haystack-pipelines/index.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: blog-post
+title: "Talk to YouTube Videos with Haystack Pipelines"
+description: Use Whisper to provide YouTube videos as context for retrieval augmented generation
+featured_image: thumbnail.png
+images: ["blog/talk-to-youtube-videos-with-haystack-pipelines/thumbnail.png"]
+featured_image_caption: Talk to YouTube Videos with Haystack Pipelines
+alt_image: Talk to YouTube Videos with Haystack Pipelines
+toc: True
```

Comment:
If you want ToC, you can update section titles as `##`. If you don't, this should be `toc: False`

Reply:
Thanks, good catch!